templates/vhost/_directories.erb

```diff
@@ -505,6 +505,10 @@
     <%- if directory['custom_fragment'] -%>
     <%= directory['custom_fragment'] %>
     <%- end -%>
+    <%- if directory['authz_core'] -%>
+      <%- scope.lookupvar('_template_scope')[:item] = directory['authz_core'] -%>
+    <%= scope.function_template(["apache/vhost/_authz_core.erb"]) -%>
```

Comment:
mhm I don't know if we can call epp templates here as well? That would allow us to add `_auth_core` as epp and not erb template. erb is considered legacy and we want to get rid of them.

Reply:
done!!